preview_generator/preview/builder_factory.py

```diff
@@ -98,7 +98,12 @@ def load_builders(self, force: bool = False) -> None:
             from preview_generator.preview.generic_preview import PreviewBuilder  # nopep8
 
             for cls in get_subclasses_recursively(PreviewBuilder):
-                self.register_builder(cls)
+                if cls.__name__ == "ImagePreviewBuilderWand":
+                    self.logger.warn(
+                        "ImagePreviewBuilderWand builder deprecated, do not register by default."
+                    )
```

Comment:
Does this mean that out of the box on a new installation, PreviewGenerator will output a warning a deprecated builder?
I think a warning should be output to the user if they can do something to fix it.

```suggestion
                    self.logger.info(
                        "ImagePreviewBuilderWand builder is deprecated and is not registered by default. Consider using ImagePreviewBuilderIMConvert instead"
                    )
```

Reply:
I don't have strong opinon on this.

Reply:
@inkhey I do not see reason not to load Wand: you are about to potentially implement regression in user implementations based on preview-generator

Reply:
after discussion with @lebouquetin, we make clear that not loading wand is not a problem, as all supported format by wand are supported by imagemagick too.